package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pooltogether/pooltogether-contracts",
-  "version": "3.4.1-alpha.5",
+  "version": "3.4.0",
```

Comment:
Unforunately, it appears that 3.4.0 was published 19 days ago:

https://www.npmjs.com/package/@pooltogether/pooltogether-contracts

So it won't be possible to re-publish this.  This will have to become 3.4.1